.github/workflows/claude-pr-creator.yml

```
@@ -35,6 +35,9 @@ jobs:
           token: ${{ steps.app-token.outputs.token }}
           persist-credentials: false
 
+      - name: Setup pnpm
+        uses: ./.github/actions/pnpm-setup
```

actionlint (1.7.7)

description is required in metadata of "pnpm-setup" action at "/home/jailuser/git/.github/actions/pnpm-setup/action.yml"

(action)